srcpkgs/ibus-array/template

```diff
@@ -1,10 +1,11 @@
 # Template file for 'ibus-array'
 pkgname=ibus-array
 version=0.2.2
-revision=1
+revision=2
 build_style=gnu-configure
 hostmakedepends="automake pkg-config python3 gettext-devel libtool"
 makedepends="ibus-devel gettext-devel sqlite-devel"
+depends="ibus"
```

Comment:
ibus-array already depends on ibus, you can confirm using `xbps-query -Rx ibus-array`.
(it should be added in the "collect rundeps" step)